WooCommerce/Classes/ViewRelated/ReusableViews/LabeledTextViewTableViewCell.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+import Yosemite
+/// A table view cell that contains a label and a text view.
+///
+final class LabeledTextViewTableViewCell: UITableViewCell {
+    struct ViewModel {
+        var text: String? = nil
+        var productStatus: ProductStatus
+        var placeholder: String? = nil
+        var textViewMinimumHeight: CGFloat? = nil
+        var isScrollEnabled: Bool = true
+        var keyboardType: UIKeyboardType = .default
+        var onNameChange: ((_ text: String) -> Void)? = nil
+        var onTextDidBeginEditing: (() -> Void)? = nil
+        var style: Style = .headline
+
+    }
+    @IBOutlet weak var productLabelHolder: UIView!
+    @IBOutlet weak var productStatusLabel: UILabel!
+    @IBOutlet var productTextField: EnhancedTextView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureLabelStyle()
+        configureBackground()
+    }
+
+    func configure(with viewModel: ViewModel) {
+        productTextField.text = viewModel.text
+        productTextField.placeholder = viewModel.placeholder
+        productStatusLabel.text = viewModel.productStatus.description
+        productTextField.isScrollEnabled = viewModel.isScrollEnabled
+        productTextField.onTextChange = viewModel.onNameChange
+        productTextField.onTextDidBeginEditing = viewModel.onTextDidBeginEditing
+        productTextField.keyboardType = viewModel.keyboardType
+        configureProductStatusLabel(productStatus: viewModel.productStatus)
```

Comment:
```suggestion
        configureProductStatusLabel(productStatus: viewModel.productStatus)
        applyStyle(style: viewModel.style)
```

It looks like this method is missing a call to `applyStyle`, so the text field isn't properly configured, and the font won't respond to dynamic text changes:

![Screen Shot 2021-09-06 at 11 11 51](https://user-images.githubusercontent.com/8739/132192282-268fab68-cae0-457f-994d-d285de2d21f2.png)

Reply:
Thanks for pointing this out! I may have missed this bit while trying to adapt the new cell to the old implementation. By your screenshot, applying the current style doesn't seem to fit the overall design and font size ( is too small ), so I'll iterate over it and find a more suitable font and size for this.